docs/guides/run-elasticsearch.rst

```diff
@@ -0,0 +1,84 @@
+Enabling Elasticsearch on the local server
+==========================================
+
+Read the Docs has been using Elasticsearch which is a platform for distributed search and analysis of data in real time. To enable the search feature on your local installation, you need to install the elasticsearch locally and run the Elastic server. 
+
+Installation has been mainly divided into following steps.
+
+1. Installing Java
```

Comment:
Isn't better to link to official resources for this step? So we don't  need to maintain this at the future or write it for every OS.

Reply:
ok @stsewd , I will do the required changes